pkg/aggregator/demultiplexer.go

```diff
@@ -54,6 +55,10 @@ type Demultiplexer interface {
 	// Main idea is to reduce the garbage generated by slices allocation.
 	GetMetricSamplePool() *metrics.MetricSamplePool
 
+	// ReconfigTimeSamplersBlocklist triggers a reconfiguration of the blocklist
+	// applied in the time samplers.
+	ReconfigTimeSamplersBlocklist(*utilstrings.Blocklist)
```

Comment:
Since it's `SetBlocklist` in dogstatsd, for consistency:

```suggestion
	SetTimeSamplersBlocklist(*utilstrings.Blocklist)
```

Reply:
Good idea, addressed.